website/docs/r/transit_secret_backend_cache_config.html.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: "vault"
+page_title: "Vault: vault_transit_secret_backend_cache_config resource"
+sidebar_current: "docs-vault-resource-transit-secret-backend-cache-config"
+description: |-
+  Configure the cache for the Transit Secret Backend in Vault.
+---
+
+# vault\_transit\_secret\_backend\_cache\_config
+
+Configured the cache for the Transit Secret Backend in Vault.
+
+## Example Usage
+
+```hcl
+resource "vault_mount" "transit" {
+  path                      = "transit"
+  type                      = "transit"
+  description               = "Example description"
+  default_lease_ttl_seconds = 3600
+  max_lease_ttl_seconds     = 86400
+}
+
+resource "vault_transit_secret_backend_cache_config" "cfg" {
+  backend = "${vault_mount.transit.path}"
+  size    = 500
+}
+
+```
+## Argument Reference
+
+The following arguments are supported:
+
+* `backend` - (Required) The path the transit secret backend is mounted at, with no leading or trailing `/`s.
```

Comment:
Should this be named `path` instead? Seems it's expected to receive the `path` attribute of other resources, but we call it `backend`.

Reply:
What happens if they do include a trailing or leading `/`? Is it conceivable that whatever it's interpolated would have that, and they wouldn't actually input that as the example shows? If they shouldn't have it, it may be worth adding a state and diff functions to the attribute.

Reply:
I agree that `path` is a bit nicer, but`backend` was chosen mainly for consistency with the other transit resource: https://github.com/terraform-providers/terraform-provider-vault/blob/754a58aa908d524e0c91136b7e65d660c1f46107/vault/resource_transit_secret_backend_key.go#L31

Regarding slashes, adding a trailing slash doesn't cause any issues... just a path name diff on the first apply, but that's it. Adding a leading slash is an entirely different path, so it would need to match the path already mounted.